torch/csrc/distributed/rpc/tensorpipe_agent.h

```diff
@@ -34,6 +43,11 @@ namespace channel {
 template <typename TBuffer>
 class Context;
 using CpuContext = Context<CpuBuffer>;
+
+#if TENSORPIPE_USE_CUDA_IPC_CHANNEL
+using CudaContext = Context<CudaBuffer>;
+#endif
```

Comment:
Same here: we should make this depend on the general availability of CUDA, not of the IPC channel.
```suggestion
#if USE_CUDA
using CudaContext = Context<CudaBuffer>;
#endif
```